core-bundle/contao/languages/en/default.xlf

```diff
@@ -275,6 +275,12 @@
       <trans-unit id="ERR.startStop">
         <source>The end time must be later than the start time.</source>
       </trans-unit>
+      <trans-unit id="ERR.noAdminEmailUrl">
+        <source>No administrator e-mail address has been set in the &lt;a href="%s" target="_top"&gt;system settings&lt;/a&gt;.</source>
+      </trans-unit>
+      <trans-unit id="ERR.noAdminEmail">
+        <source>No administrator e-mail address has been set in the system settings.</source>
```

Comment:
This is a bit misleading, because the address might as well be defined in the container configuration. Would it be an option to just say _No administrator e-mail address has been set yet_?

Reply:
In that case the system settings module will show you that it is set via the configuration.

Reply:
You mean the system settings that the user does not have access to?

Reply:
You mean in the case where:

* The system settings are not enabled for the user.
* The administrator email address is set via the container configuration.

In that case the user will have to contact the system administrator anyway and will then either:

* Have access to the system settings, where it is explained to them that the setting is set via the container configuration.
* Or the system administrator knows where to change the administrator email address, since they are the one who set it in the container configuration in the first place.

Reply:
No, that is not what I meant. If a use cannot access the system settings, we should not tell them "it has not been set in the system settings", because that will make them search the system settings trying to fix the issue. We should say "has not been set" or "has not been set, please contact the site administrator" instead.

Reply:
Alright, so

> There is no administrator e-mail address set, please contact the site administrator.

?